azure-pipelines-e2e-tests-template.yml

```diff
@@ -8,6 +8,13 @@ parameters:
 - name: forwardCompatibleRelease
   type: string
   default: ''
+- name: tests.enableForwardCompatibleTests
+  type: boolean
+  default: true
+- name: tests.enableBackwardCompatibleTests
+  type: boolean
+  default: true
+  
```

Comment:
I don't think this is the proper way to add defaults to the `tests` parameter.  May have to readd all the true/false settings from the caller..

Reply:
done

Reply:
Although the tests do seem to be behaving as expected with the previous change

Reply:
That may or may not have been working maybe as a side effect of how booleans are interpreted and what their non specified default values are interpreted by the system. I think another way to check if it was really working was to change that `true` to `false` as the default and see if the other versions were behaving as expected.